ccdImports/configFiles/CCD_Probate_Backoffice/AuthorisationCaseEvent.json

```diff
@@ -132,6 +132,8 @@
   {"CaseTypeID": "GrantOfRepresentation", "CaseEventID": "deathRecordVerificationFailed", "UserRole": "caseworker-probate-issuer", "CRUD": "R"},
   {"CaseTypeID": "GrantOfRepresentation", "CaseEventID": "levSelectFromMultipleRecords", "UserRole": "caseworker-probate-issuer", "CRUD": "CRUD"},
   {"CaseTypeID": "GrantOfRepresentation", "CaseEventID": "manuallyVerifyDeathRecord", "UserRole": "caseworker-probate-issuer", "CRUD": "CRUD"},
+  {"CaseTypeID": "GrantOfRepresentation", "CaseEventID": "makeCaseDormant", "UserRole": "caseworker-probate-issuer", "CRUD": "R"},
```

Comment:
these 2 need to be applied for all the caseworker-probate-* roles except for the ones you have already set below

Reply:
Added